R/ruby/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ruby"
+version = v"2.6.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource(
+        "https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.6.tar.gz",
+        "364b143def360bac1b74eb56ed60b1a0dca6439b00157ae11ff77d5cd2e92291",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ruby-2.6.6/
```

Comment:
Minor suggestion:
```suggestion
cd $WORKSPACE/srcdir/ruby-*/
```
This way, updating to a newer version will produce a smaller diff (and this also removes the risk of forgetting to update the version here, and the resulting extra roundtrip to fix the package ;-)